### PR TITLE
Fix for non-millisecond timestamps

### LINKED ISSUE #560: CloudDFG timestamp bug
If no decimal for seconds.
ArgumentError: Unable to parse string "2020-07-27T00:27:34Z" using format dateformat"yyyy-mm-ddTHH:MM:SS.ssszzz". Unable to parse date time. Expected directive Delim(.) at char 20

### PR DESCRIPTION
**Pull request by GearsAD**
Fixes #560

### CODE REVIEW

**Comment by dehann** [comment]
Super cool!

File: src/services/Serialization.jl [hunk 1]
```diff
@@ -30,6 +30,9 @@ function unpackVariable(dfg::G,
         unpackBigData::Bool=true)::DFGVariable where G <: AbstractDFG
     @debug "Unpacking variable:\r\n$packedProps"
     label = Symbol(packedProps["label"])
+    # Make sure that the timestamp is correctly formatted with subseconds
+    packedProps["timestamp"] = replace(packedProps["timestamp"], r":(\d)(\d)(Z|z|\+|-)" => s":\1\2.000\3")
```
Comment (Affie) [inline src/services/Serialization.jl:34]: Is the correct format not enforced by my change in ln 11:
`props["timestamp"] = Dates.format(v.timestamp, "yyyy-mm-ddTHH:MM:SS.ssszzz")`

As far as I could tell this was only a bug in CGDFG and worked correctly with forcing the format as above

**Comment by dehann** [comment]
Don't think this method actually works, see here https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/582#issuecomment-671976222

File: src/services/Serialization.jl (hunk 1 above)
Comment (dehann) [reply src/services/Serialization.jl:34]: 11?

Comment (dehann) [reply src/services/Serialization.jl:34]: Nope, see here: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/582#issuecomment-671976222

Comment (Affie) [reply src/services/Serialization.jl:34]: Not the same thing

Comment (Affie) [reply src/services/Serialization.jl:34]: As far as I could tell it was working in v0.9, just not in the cloud serialization, which is separate.

**Pull request merged**